roles/ceph-purge/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: check if ceph is on the system already
+  stat: path=/var/lib/ceph/osd/
+  register: ceph_on_system
+
+- name: clear osd devices
+  shell: dd if=/dev/zero of={{ item }} bs=1M count=512
+  with_items: ceph_osd.devices
+  when: ceph_on_system.stat.exists == false or
+        ceph_purge.destroy_devices_no_matter_what
+
+- name: clear journal devices
+  shell: dd if=/dev/zero of={{ item }} bs=1M count=512
```

Comment:
This will not erase a GPT partition table since it keeps a backup and the beginning and end of disk. Looks like sgdisk --zap will do the trick.